ConfigurationSystem/private/ConfigurationData.py

```diff
@@ -363,10 +361,9 @@ def __backupCurrentConfiguration( self, backupName ):
   def writeRemoteConfigurationToDisk( self, backupName = False ):
     configurationFile = os.path.join( DIRAC.rootPath, "etc", "%s.cfg" % self.getName() )
     try:
-      fd = open( configurationFile, "w" )
-      fd.write( str( self.remoteCFG ) )
-      fd.close()
-    except Exception, e:
+      with open( configurationFile, "w" ) as fd:
+        fd.write( str( self.remoteCFG ) )
+    except Exception as e:
       gLogger.fatal( "Cannot write new configuration to disk!",
                      "file %s" % configurationFile )
       return S_ERROR( "Can't write cs file %s!: %s" % ( configurationFile, str( e ) ) )
```

Comment:
I am willing to produce this 3 lines function as I did it already twice, but where should it go?